src/minicraft/level/tile/TreeTile.java

```diff
@@ -102,8 +102,8 @@ public void hurt(Level level, int x, int y, int dmg) {
 
 		level.add(new TextParticle("" + dmg, x*16+8, y*16+8, Color.RED));
 		if (damage >= treeHealth) {
-			level.dropItem(x*16+8, y*16+8, 1, 2, Items.get("Wood"));
-			level.dropItem(x*16+8, y*16+8, 1, 2, Items.get("Acorn"));
+			level.dropItem(x*16+8, y*16+8, 1, 3, Items.get("Wood"));
+			level.dropItem(x*16+8, y*16+8, 0, 2, Items.get("Acorn"));

```

Comment:
I suppose it makes sense to allow zero, since people probably end up with a vast quantity of acorns.